### ISSUE TITLE
Вопрос о 2 тестировании в МГТУ

### ISSUE THREAD

**Issue by Avonna** [Kiser1983/Antonov_VKR_DS#1]
Здравствуйте.  
Прошу прощения за беспокойство. Я сейчас обучаюсь на курсе Data Science в МГТУ и прохожу второе тестирование. Там есть задание с датасетом: #Разработать и обучить нейронную сеть на TensorFlow для решения задачи классификации рукописных цифр. Набор данных: MNIST, источник: from keras.datasets import mnist 
 
#В нейронной сети должно быть 3 Dense-слоя: два скрытых слоя, состоящих из 128 нейронов каждый, и выходной слой. Активационные функции на скрытых слоях relu, на выходном - softmax. 
 
#В качестве ответа необходимо указать количество оптимизируемых параметров (Trainable params) в виде целого числа 
#и прикрепить к ответу подробное решение с кодом в виде файла или архива с файлами.

Хотела спросить, вдруг у вас осталось решение. Правильный ли ответ: Total params: 147,402
Trainable params: 147,402
Non-trainable params: 0
__________________________Благодарю.

**Comment by Kiser1983** [comment]
у нас ответ : 118282
 
--
Сергей Антонов
Отправлено из Почты  Mail.ru

**Comment by Kiser1983** [comment]
ответ 118282
[Test2_Antonov_ver2.zip](https://github.com/Kiser1983/Antonov_VKR_DS/files/8884566/Test2_Antonov_ver2.zip)

**Comment by Avonna** [comment]
Спасибо большое! Вы - чудо! Поняла, где ошиблась. Благодарю Вас от всей души! Удачи Вам и всего самого хорошего. 
P.S. Защищаться сложно было?

**Comment by Kiser1983** [comment]
не сложно, главное понимать как работает код в вкр
